src/core/appdir.cpp

```diff
@@ -392,8 +393,8 @@ namespace linuxdeploy {
                             return false;
                         }
 
-                        static auto isInExcludelist = [](const bf::path& fileName) {
-                            for (const auto& excludePattern : generatedExcludelist) {
+                        static auto isInExcludelist = [](const bf::path& fileName, const std::vector<std::string> &excludeList) {
```

Comment:
Just capture `this`, then you can access member variables.

Reply:
Its passing in the exclude list because `isInExcludelist` is used to check both lists (command line and built-in), or is there a different variable that could be captured I'm not seeing?